test/styles/RRR.R

```diff
@@ -39,38 +48,110 @@ fun_call(argument1,
 fun_call(
     argument1,
     argument2
-    )
+)
 
 ## 3
 fun_call(parameter = (
     stuff
-    ),
-         argument)
+),
+argument)
 
 ## 4
 fun_call(parameter = fun_argument(
-             stuff
-    ),
-         argument)
+    argument1
```

Comment:
Ok. I think I finally started getting it.  So `arguments-newline` take precedence everywhere right? So if's only when you keep an argument on the same line then `ess-indent-from-outer-parameter` kicks in. That's indeed consistent.